brigade/core/__init__.py

```diff
@@ -52,9 +53,10 @@ class Brigade(object):
         config (:obj:`brigade.core.configuration.Config`): Configuration parameters
     """
 
-    def __init__(self, inventory, dry_run,
-                 config=None, config_file=None):
+    def __init__(self, inventory, dry_run, config=None, config_file=None,
+                 available_connections=None):
         self.inventory = inventory
+        self.inventory.brigade = self
```

Comment:
Shouldn't this be `self.inventory._brigade = self`?

Reply:
I think we should document it and make it official. It's useful to access other hosts' data and the configuration.

Reply:
@dbarrosop Yes, but this line 59 needs to be fixed correct and not be `self.inventory.brigade` and instead be `self.inventory._brigade` (i.e it is missing the underscore)?

Reply:
why? if we want to make it official we shouldn’t mark it as private.

Reply:
My error, what you have here...should be good.

Reply:
cool, thanks. Let me know if you want me to fix netmiko’s connection task. Otherwise I will fix napalm’s and the documentation and merge later today.